website/src/pages/index.module.css

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap');
+@import url('https://db.onlinewebfonts.com/c/00d5a78493aed4b11e2584ad7cceee49?family=Gilroy-ExtraBold');
+
+html {
+  background-color: var(--ifm-background-color);
+}
+
+html[data-theme='dark'] {
+  --ifm-background-color: #3B4252;
+  --ifm-hero-background-color: #2E3440;
+}
+html[data-theme='light'] {
+  --ifm-background-color: white;
+  --ifm-hero-background-color: whitesmoke;
+}
+
+.heroBanner {
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+
+  height: calc(100vh - 110px);
```

Comment:
This is too much, can we just remove this CSS rule - the website looks fine without it. I'd rather base the height on the content than the view dimensions.